.rubocop_todo.yml

```diff
@@ -65,3 +65,11 @@ Style/PredicateName:
   Exclude:
     - 'spec/**/*'
     - 'lib/foundation_rails_helper/form_builder.rb'
+
+# Offense count: 11
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, ConsistentQuotesInMultiline.
+# SupportedStyles: single_quotes, double_quotes
+Style/StringLiterals:
+  Exclude:
+    - 'spec/support/mock_rails.rb'
```

Comment:
@dsandstrom StringLiterals is an autofixable style: you can remove this paragraph and run `rubocop --auto-correct` or just `rubocop -a` to have it automatically fix the quotes.